doc-redirects.properties

```diff
@@ -45,64 +45,13 @@
 
 # Temporary redirect until https://github.com/OpenLiberty/docs/issues/951 is merged
 /docs/latest/why-open-liberty.html=/
-/docs/20.0.0.10/why-open-liberty.html=/
```

Comment:
Do we know what was the purpose of this redirect?

- Are there external websites using the `/docs/<version>/why-open-liberty.html` link?
- Did this page `why-open-liberty.html` once live in production? 
     - I was not able to find the page after using `grep` on the entire website application files.
- Why did we keep adding more redirects?
- Why did we decide to only start at `20.0.0.10` and not include all the other Liberty versions? 
     - This link gives redirects to the 404 page https://openliberty.io/docs/20.0.0.9/why-open-liberty.html

I think the cleaner way is to try using a regular expression and consolidate the redirects into one line, if we discover this redirect must remain.

@mbroz2 @steven1046, do you recall why we have this redirect? Can it be removed?
I found https://github.com/OpenLiberty/openliberty.io/issues/1897 but it is not clear why the temporary redirect is needed.

Reply:
This was a requirement raised by @gcharters to be used in marketing material (aka, the link is used externally).  
The link needed to be provided before the content was available, hence the redirect.  I believe the content was going to be provided via https://github.com/OpenLiberty/docs/issues/951 but @dmuelle would likely know better (along with the state of the work.)

Reply:
That doc has been iceboxed because I could never get approval for it. I made all the changes from. @mbroz2 last review and had reached out for further reviews from graham and alasdair but seems to have got lost in the shuffle. The draft is viewable on the draft site at https://draft-openlibertyio.mybluemix.net/docs/latest/why-open-liberty-draft.html. It's been sitting long enough now that it might need updates but it should be pretty much ready to publish anytime

Reply:
reached out to @gcharters via slack to confirm whether the original link specified `latest` or a version number. If we can't confirm, I'll edit the PR to leave whyliberty redirects in the doc